maintenance_changing_maintainers.es.Rmd

```diff
@@ -1,59 +1,61 @@
-# Changing package maintainers {#changing-maintainers}
+# Cambio de mantenedores de paquetes {#changing-maintainers}
 
-```{block, type='summaryblock'}
-This chapter presents our guidance for taking over maintenance of a package.
+```{block, type="summaryblock"}
+Este capítulo presenta nuestra guía para asumir el mantenimiento de un paquete.
 ```
 
-## Do you want to give up maintenance of your package?
+## ¿Quieres dejar el mantenimiento de tu paquete?
 
-We have a call for contributors section in our newsletter that comes out every two weeks. The section is called _Call For Contributors_. In that section we highlight packages looking for new maintainers. If you are looking to leave the role of maintainer of your package, get in touch with us and we can highlight your package in our newsletter. We've been very successful thus far, finding new maintainers for six out of six packages.
```

Comment:
> We've been very successful thus far, finding new maintainers for six out of six packages.

Perhaps it might be better to remove the precise number of packages not to have to update this every time a package gets a new maintainer through the newsletter?

Reply:
oops indeed! do you want to make this change with a PR? thank you in any case!